dill/_dill.py

```diff
@@ -1449,6 +1621,10 @@ def save_module(pickler, obj):
             pickler.save_reduce(_import_module, (obj.__name__,), obj=obj,
                                 state=_main_dict)
             log.info("# M1")
+        elif PY3 and obj.__name__ == "dill._dill":
```

Comment:
What is the use case for a module `dill._dill` here and just above?  Is it better handled by moving the relevant objects into the module `dill`?

Reply:
The issue was that, without the if condition, dill would pickle an entire copy of the `dill` module instead of treating it as a global. I'll look into it and see why that is, but this was an easy way to get around that.

Reply:
Was there any follow-up on looking into alternatives here?  It is acceptable as is, I guess, as it's probably needed when we start pickling objects defined in `dill._dill` that are not also added to the `dill` namespace.

Reply:
...I assume that the conclusion here is to just leave it as is.

Reply:
@anivegesana: I'm marking this as resolved... but please comment here if it is not.